src/engraving/dom/spanner.cpp

```diff
@@ -225,7 +225,7 @@ Sid SpannerSegment::getPropertyStyle(Pid pid) const
 
 PropertyFlags SpannerSegment::propertyFlags(Pid pid) const
 {
-    if (EngravingItem* e = const_cast<SpannerSegment*>(this)->propertyDelegate(pid)) {
+    if (EngravingObject* e = const_cast<SpannerSegment*>(this)->propertyDelegate(pid)) {
```

Comment:
Doesn't this just duplicate the logic in EngravingObject now? In that case, the entire override can be removed. Otherwise, I do think we can do without the const_cast. 
(Same for some other methods in this file)

Reply:
Yes, meant to tidy this and forgot!